### ISSUE TITLE
ID/Relay server dialog not shown

### ISSUE THREAD

**Issue by rustdesk** [rustdesk/rustdesk#726]
> After installing the server on the Windows server in the pure intranet environment of rustdesk, the client cannot input the IP of the relay server, and the input window will disappear when the input window is opened. Both the client and the server are downloaded from the official website.

**Comment by cerri** [comment]
Yes, but if we want to use a selfhosted environment, the client setup cannot be done.

**Comment by rustdesk** [comment]
Windows binary updated ,https://github.com/rustdesk/rustdesk/releases/tag/1.1.9
